hugegraph-pd/hg-pd-dist/src/assembly/static/bin/start-hugegraph-pd.sh

```diff
@@ -115,7 +115,7 @@ echo "Starting HugeGraphPDServer..."
 
 # Turn on security check
 exec ${JAVA} ${JAVA_OPTIONS} -jar -Dspring.config.location=${CONF}/application.yml \
```

Comment:
could also add `process name` param for it..... refer server/toolchain/computer, so as `store`

Reply:
add `process name` param for `pd`

`store` already added